src/AppWrapper.tsx

```diff
@@ -79,9 +78,11 @@ export default function AppWrapper(): JSX.Element {
     }, []);
 
     return (
-        <div className='App'>
+        <div className='flex min-h-screen flex-col place-items-center'>
```

Comment:
❓ Do we want to use a semantic HTML tag here `main`

Reply:
Yes actually, that is something I did not consider.